internal/app/handler/url.go

```diff
@@ -0,0 +1,10 @@
+package handler
```

Comment:
```suggestion
package dto
```
It should be in the `dto` package.